src/app/public/testing/filter/filter-fixture-summary.ts

```diff
@@ -0,0 +1,43 @@
+import {
+  DebugElement
+} from '@angular/core';
+
+import {
+  ComponentFixture
+} from '@angular/core/testing';
+
+import {
+  SkyAppTestUtility
+} from '@skyux-sdk/testing';
+
+/**
+ * Provides information for and interaction with a SKY UX filter button component.
```

Comment:
Provides information for and interaction with a SKY UX `filter summary` component.